tests/testthat/test-az_heat.R

```diff
@@ -45,7 +45,7 @@ test_that("data is in correct format", {
     res_default <- az_heat()
   })
   expect_s3_class(res_default, "data.frame")
-  expect_equal(nrow(res_default), 28)
+  expect_equal(nrow(res_default), nrow(station_info))
```

Comment:
Nice edit. This will make for an easier time when stations are added to or removed from the network.